framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -74,6 +77,7 @@
 import com.sun.source.tree.Tree;
 import com.sun.source.tree.VariableTree;
 import com.sun.source.util.TreePath;
+import com.sun.tools.javac.code.Symbol.ClassSymbol;
```

Comment:
Given a `ClassTree`, is it possible to obtain the fully qualified name of the class? In case that is possible I could remove this dependency and clean up some code.